OracleUnifiedDirectory/dockerfiles/12.2.1.4.0/install/oraInst.loc

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Property and configuration files like `oraInst.loc` don't need copyright or license headers.

Reply:
Removed.